source/support.tex

```diff
@@ -729,6 +729,9 @@
 
   // \ref{numeric.limits}, class template \tcode{numeric_limits}
   template<class T> class numeric_limits;
+  template<class T> class numeric_limits<const T>;
```

Comment:
newline between the primary and these partials?

Reply:
Sure.